app/models/publication.rb

```diff
@@ -216,6 +223,10 @@ def confirmed_users
     users.where(authorships: { confirmed: true })
   end
 
+  def activity_insight_file_uploader
```

Comment:
Sorry, I have second thoughts on the name of this method - but only because there's a class in this application named `ActivityInsightFileUploader`, which is not at all what this method returns. I'd be less confused if it was called something like `activity_insight_upload_user`.